### PR TITLE
feat(web): add table with meters

### PR DESCRIPTION
**Pull request by eTallang**
OMEGA-33

## What is the reason for this change?

A new table with meterpoints is required for the restructure of Min Side.

### What is the expected result?

A new table should be visible with a list of all meterpoints.

## How can it be tested and verified?

- [ ] Check that the implementet table looks and behaves as designed in [Figma](https://www.figma.com/file/sJHoScmQRq0hlr6AcHI9WB/Elvia.no?node-id=9560%3A55962).
- [ ] Check that the table is keyboard accessible.
- [ ] Check that clicking a row directs to the correct meterpoint page.
- [ ] Check that the correct meterpoints are visible in the table.
- [ ] Check that the table is responsive according to the designs.

### CODE REVIEW

File: packages/web/src/templates/Meterpoints/Meterpoints.module.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
+@import '../../theme';
+@import '@elvia/elvis/src/utilities/breakpoints.scss';
+
+.tableContainer,
+.table {
```
Comment (lobunto) [inline packages/web/src/templates/Meterpoints/Meterpoints.module.scss:5]: Is this table styling special for the meteringpoint list?

File: packages/web/src/templates/Meterpoints/Meterpoints.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    if (event.key === 'Enter') {
+      goToMeterpoint(id)
+    }
+  }
+
+  const getAddress = (meteringPoint: MeteringPoint) => {
+    const address = meteringPoint.address
+    return `${address.streetName} ${address.buildingNumber}${address.letter ??
+      ''}`
+  }
```
Comment (lobunto) [inline packages/web/src/templates/Meterpoints/Meterpoints.tsx:37]: This is great! 👍

File: packages/web/src/templates/Meterpoints/Meterpoints.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+  const onKeyDown = (
+    event: React.KeyboardEvent<HTMLTableRowElement>,
+    id: string
+  ) => {
+    if (event.key === 'Enter') {
+      goToMeterpoint(id)
+    }
+  }
```
Comment (lobunto) [inline packages/web/src/templates/Meterpoints/Meterpoints.tsx:31]: This is great! 👍

File: packages/web/src/pages/minside.tsx [hunk 4]
```diff
@@ -87,8 +88,8 @@ const Minside: React.FC<PageContext> = props => {
           path="/minside/kundeforhold/:customerId"
           component={CustomerDetail}
         >
-          <Redirect path="/" from="/" to="malerpunkt" noThrow />
-          <PrivateRoute path="malerpunkt" component={Meterpoint} />
+          <Redirect path="/" from="/" to="malere" noThrow />
+          <PrivateRoute path="malere" component={Meterpoints} />
```
Comment (lobunto) [inline packages/web/src/pages/minside.tsx:92]: Will this route be default even if there is only one facility?

**Review by lobunto — COMMENTED** [review]
LGTM. Got a question tho about the route for meteringpoints

File: packages/web/src/pages/minside.tsx (hunk 4 above)
Comment (eTallang) [reply packages/web/src/pages/minside.tsx:92]: Thats a good question. I haven't heard anything else, but I'll check with Malin.

Comment (eTallang) [reply packages/web/src/pages/minside.tsx:92]: The designers have discussed it, and if we are to have an exception for customers with one meteringpoint, it's gonna result in three different states for the min side landing page. So lets take a closer look at this later :slightly_smiling_face:

**Comment by ragnhildlarsen** [comment]
I think you need to ensure that the meters shown is the one connected to the customer relation that you are looking at. 

E.g. Per Hansen only have 2 meterpoints himself, and the rest in the list are related to the other customer relationships. 

Screenshot 2020-09-11 at 15 28 06

**Comment by eTallang** [comment]
> I think you need to ensure that the meters shown are connected to the customer relation that you are looking at.
> 
> E.g. Per Hansen only have 2 meterpoints himself, and the rest in the list are related to the other customer relationships.
> 
> Screenshot 2020-09-11 at 15 28 06

A good catch! Fixing it now.

**Review by lobunto — APPROVED** [review]

**Pull request merged**